### PR TITLE
Added ConstantBiasScale

### PR DESCRIPTION
**Pull request by gkmotu**

### CODE REVIEW

File: ConstantBiasScale.osl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+   string label    = "ConstantBiasScale (vector)",
+   string category = "Math\Vector" ]]
+(
+	vector in = 0.5,
+	float bias = 0.5,
+	float scale = 2.0,
+	output vector Out = 0.0,
+)
+{
+	Out = (in+bias)*scale;
```
Comment (MasterZap) [inline ConstantBiasScale.osl]: Wouldn't you normally want to REMOVE a bias (subtract it)?

**Comment by MasterZap** [comment]
You should update your repo (try https://upriver.github.io/) and put stuff in the appropriate directory
(A subdiretory with your name i.e. under OSL/ADN-User Submitted/gkmotu/ )

File: ConstantBiasScale.osl (hunk 1 above)
Comment (gkmotu) [reply ConstantBiasScale.osl]: The cases I have stumbled across so far works with add and they work in my implementation of Flowmap. I guess, its (input+-0.5)*2, I guess minus always overrules plus. So user can put a negative number anyways as I gather it. But you could be right ~

Comment (MasterZap) [reply ConstantBiasScale.osl]: I dunno, it doesn't matter really. To me I'm thinking "here's a vector with a bias of 0.5" and if I have a shader that claims to remove that bias (and it calls the parameter bias), I then personally kinda expected to put in 0.5 to remove a 0.5 bias, not -0.5 .... but I agree this is.... well... open to interpretation :)

**Pull request merged**